packages/component/src/SendBox/ConnectivityStatus.js

```diff
@@ -1,7 +1,7 @@
 /* eslint react/no-unsafe: off */
```

Comment:
Could you do a broad search on keyword react/no-unsafe and remove them? And file bugs for those we cannot remove as we didn't migrate yet.